src/Compilers/CSharp/Portable/Symbols/MemberSymbolExtensions.cs

```diff
@@ -545,22 +545,28 @@ internal static bool IsExplicitInterfaceImplementation(this Symbol member)
             }
         }
 
-        internal static bool IsPartialMethod(this Symbol member)
+        internal static bool IsPartialMember(this Symbol member)
```

Comment:
I did consider introducing some new members on a common base type, or introducing an `IPartialMemberSymbol` interface or something similar. But ultimately I think extension methods with type tests is an appropriate level of abstraction for this feature.

I am meaning to add a debug assertion that these methods are only used with original definition symbols. I'll try and add a check for that in future PR.